src/tests/unit/tests/DetailsView/components/quick-assess-to-assessment-dialog.test.tsx

```diff
@@ -64,12 +67,9 @@ describe('QuickAssessToAssessmentDialog', () => {
     test('onclick: continue to assessment', async () => {
         props.isShown = true;
         const eventStub = {} as SupportedMouseEvent;
-        const testSubject = shallow(
-            <QuickAssessToAssessmentDialog {...props}></QuickAssessToAssessmentDialog>,
-        );
-        const onClick = testSubject.find(PrimaryButton).prop('onClick');
 
-        await onClick(eventStub as any);
+        render(<QuickAssessToAssessmentDialog {...props}></QuickAssessToAssessmentDialog>);
```

Comment:
<QuickAssessToAssessmentDialog {...props}></QuickAssessToAssessmentDialog> do we need like this?
since we dont have any child components, dont need like above it can be simply
<QuickAssessToAssessmentDialog {...props} />